frontend/src/features/admin-form/create/builder-and-design/BuilderAndDesignContent/StartPageView.tsx

```diff
@@ -0,0 +1,44 @@
+import { useMemo } from 'react'
+
+import { FormAuthType } from '~shared/types'
+
+import { FormBannerLogo } from '~features/public-form/components/FormStartPage/FormBannerLogo'
+import { FormHeader } from '~features/public-form/components/FormStartPage/FormHeader'
+import { useFormBannerLogo } from '~features/public-form/components/FormStartPage/useFormBannerLogo'
+import { useFormHeader } from '~features/public-form/components/FormStartPage/useFormHeader'
+
+import { useCreateTabForm } from '../useCreateTabForm'
+import { startPageDataSelector, useDesignStore } from '../useDesignStore'
+
+export const StartPageView = () => {
```

Comment:
we should probably reuse public form's start page component so to maintain single source of truth, and also so we can be sure the preview is accurate